content/english/blog/road_to_ubiquity.md

```diff
@@ -0,0 +1,75 @@
+
+In the current ecosystem, if you want edge functions written in WebAssembly, you'll have to use _Fastly_ or _CloudFlare_'s SDK. If you want workflows to chain WebAssembly functions, you'll need to use _Atmo_'s SDK. If you want self-contained, freestanding microservices written in WebAssembly, you could use the _Spin_ SDK. If you want portable distributed actors loosely coupled to capability contracts, you may want to use the _wasmCloud_ SDK.
+
+## Embracing the Road to Ubiquity
+As the WebAssembly community and ecosystem moves forward, we (wasmCloud and Cosmonic) need to continue to move at the vanguard of this momentum, so that we're adapting our foundations to incorporate new power, features, and flexibility as it becomes available and more mature.
+
+### The Developer SDK
+The wasmCloud developer experience today is very much influenced by the notion that we originally wanted there to be no measurable difference between consuming [first party capability providers](https://wasmcloud.dev/app-dev/std-caps/) and consuming third party providers.
+
+However, since everyone making an application or platform decision at this level will be opting into the vendor's SDK, it is necessary for us to make this SDK as optimized for developers as as possible, making the following first-party activities and capabilities "brain-dead simple" _by default_:
+
+* HTTP Server
+* HTTP Client
+* Message Broker
+* Blob Store (e.g. S3)
+* Key Value
+* SQL DB
+* Actor-to-Actor Calls
+
+This wasmCloud developer SDK will be a thin wrapper on top of the generated interface code designed to smooth and optimize the developer experience and ease of use. Using this SDK will insulate developers from the churn and change happening at the engine and specification level. As `wit` and the component model mature, we'll simply integrate that work into our SDK, sparing developers from as many breaking changes as possible.
+
+### The Component Model
+Once WebAssembly is boring and ubiquitous and tooling exists for all manner of scenarios and the component model has become mature, developers will be able to rapidly adapt their existing code bases to any number of platform providers because consuming them becomes just a matter of generating client code against wit models.
+
+It's worth noting that practically speaking, most developers won't be doing this. The "easy default" case will likely be that developers will choose to use the ready-made, shrink-wrapped SDK of whichever platform they choose to run their code on.
+
+However, being built on the component model gives us the ability to rapidly adapt to any changes in the community and, more importantly, assert that there is _no vendor lock-in for consuming wasmCloud and Cosmonic services_, only the choice of relying on any of the portable component modules that represent the developer-facing contract for capability providers.
```

Comment:
Slight change suggestion: "no vendor lock in for consuming wasmCloud based services like Cosmonic". 

Once again, just softening to not make this a "Cosmonic-sponsored™ blog post"